### PR TITLE
Fix C4459 warning in custom_op_lite.h

### PR DESCRIPTION
**Pull request by skyline75489**
Fix C4459: declaration of 'api' hides global declaration.

The existing `api` is `Ort::api` defined in `onnxruntime_api.h`. Renaming the parameter name to avoid collision.

### CODE REVIEW

**Review by xadupre — DISMISSED** [review]

File: include/custom_op/custom_op_lite.h [hunk 1]
```diff
@@ -16,19 +16,19 @@ namespace Custom {
 
 class OrtKernelContextStorage : public ITensorStorage {
  public:
-  OrtKernelContextStorage(const OrtW::CustomOpApi& api,
+  OrtKernelContextStorage(const OrtW::CustomOpApi& api_,
```
Comment (wenbingl) [inline include/custom_op/custom_op_lite.h:19]: the last _ usually means it's a member of a class, can you change it to something like custom_op_api?

Comment (skyline75489) [reply include/custom_op/custom_op_lite.h:19]: sure, will do

Comment (skyline75489) [reply include/custom_op/custom_op_lite.h]: done

**Comment by skyline75489** [comment]
The CI is green. I think it's ready.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**